docs/guides/migrating.md

```diff
@@ -131,7 +131,7 @@ const Box = styled('div')(transition)
 // v5
 const Box = styled('div')(
   system({
-    transition: true,
+    transform: true,
```

Comment:
Here `transition` is correct – it's used for both the component prop name *and* the CSS `transition` property with the shortcut syntax here

Reply:
Ohhh I see. Wow maybe it’s just me but I got really confused there. Thanks for the clarification.

Reply:
It might not be the clearest example, so maybe it's worth using a different CSS property 🤔

Reply:
yeah, something like `whiteSpace` or `flexGrow`, I don't know...